index.html

```diff
@@ -21,6 +21,10 @@
     <br>
     <a style="margin-right: 1em;" href="https://github.com/Bitcoin-ABC/bitcoin-abc">Source Code</a>
     <a style="margin-left: 1em;" href="https://download.bitcoinabc.org/">Binaries</a>
+    <br>
+    <a style="margin-right: 1em;" href="https://hub.docker.com/r/zquestz/bitcoin-abc">Docker</a>
+    <a style="margin-left: 1em; margin-right: 1em;" href="https://aur.archlinux.org/packages/bitcoin-abc-qt/">Arch Linux</a>
```

Comment:
Rather than picking one of the available packages, we can have it point to the filtered search result of all of them: https://aur.archlinux.org/packages/?K=bitcoin-abc

Reply:
The only downside is anyone can push an aur package, and this opens us up to someone posting poisoned abc packages.